app/src/main/java/org/fossasia/openevent/activities/TracksActivity.java

```diff
@@ -44,6 +54,9 @@
     private String searchText = "";
 
     private SearchView searchView;
+    private RecyclerView recyclerView;
+    static String[] title = { "DevOps", "General", "OpenTech", "Web","Python", "Mozilla", "Exhibition"};
+    private TextView reset;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'reset' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406771674/issues/source?bid=3233850&fileBranchId=3233908#l59)